### PR TITLE
#163876985 create dashboard mini reports for the admin

### PR DESCRIPTION
**Pull request by byarustev**
#### What does this PR do?
- Adds a dashboard and a mini report to the dashboard for vendor performance

#### Description of Task to be completed?

- create a component for the vendor performance report
- create action and reducer to get data from rest endpoint
- write unit tests for the above components

#### How should this be manually tested?

- check out to this branch 
- run `npm install`
- the left menu item should have the dashboard active by default 
- vendor performance graph should be visible on the dashboard 

#### What are the relevant pivotal tracker stories?
- [#163876985](https://www.pivotaltracker.com/story/show/163876985)

#### Gif 
eatsdashboard

### CODE REVIEW

File: client/src/actions/dashboardAction.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import axios from 'axios';
+
+import {
+  FETCH_VENDOR_COLLECTED_ORDERS,
+  FETCH_VENDOR_UNCOLLECTED_ORDERS,
+  FETCH_VENDOR_CANCELED_ORDERS,
+  VENDOR_PERFORMANCE_LOADING,
+  FETCH_VENDOR_PERFORMANCE_FAILED, FETCH_VENDOR_PERFORMANCE_DATES,
```
Comment (SoniaRMK) [inline client/src/actions/dashboardAction.js]: Let each action be on its own line

File: client/src/actions/dashboardAction.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  FETCH_VENDOR_NAME, FETCH_VENDOR_RATINGS,
+} from "../actions/actionTypes";
+
+export const getKeyValues= (payload, key) =>{
+  let result=[];
+  payload.map(data=>{
```
Comment (SoniaRMK) [inline client/src/actions/dashboardAction.js]: Kindly leave a space between `data` and `=>` and `{`

**Comment by SoniaRMK** [comment]
It looks like if a user is not an admin, they can't see the details of the graphs.

**Comment by byarustev** [comment]
> It looks like if a user is not an admin, they can't see the details of the graphs.

I have move it to the admin menu since its where I should belong

**Pull request closed without merging**